charts/sda-svc/templates/auth-deploy.yaml

```
@@ -194,6 +198,13 @@ spec:
                   - key: {{ required "The name of the JWT signing key is needed" .Values.global.auth.jwtKey }}
                     path: {{ .Values.global.auth.jwtKey }}
       {{- end }}
+      - name: c4gh
+        - secret:
+          defaultMode: 0440
+          secretName: {{ required "A secret for the C4GH public key is needed" .Values.global.c4gh.publicFile }}
+          items:
+            - key: {{ .Values.global.c4gh.publicFile }}
+              path: {{ .Values.global.c4gh.publicFile }}
```

```suggestion
      - name: c4gh
        - secret:
          defaultMode: 0440
          secretName: {{ required "A secret for the C4GH public key is needed" .Values.global.c4gh.secret }}
          items:
            - key: {{ required "The C4GH public key is needed" .Values.global.c4gh.publicFile }}
              path: {{ .Values.global.c4gh.publicFile }}
```

The `secretName` must be an actual secret or the deployment will fail.
We also must make sure the public key is set

---

Shouldn't it be then `secretName` coming from here?: https://github.com/neicnordic/sensitive-data-archive/blob/35cc0d4ddd907f5dede9651ce6c4c2d3f1f31f1f/.github/integration/scripts/charts/values.yaml#L48
I can't narrow down why the key from above is still missing in the integration test.

---

See latest comment